Makefile

```diff
@@ -127,6 +127,18 @@ vet: gowork ## Run go vet against code.
 	go vet ./...
 	go vet ./api/...
 
+.PHONY: tidy
+tidy:
+	go mod tidy; \
+	pushd "$(LOCALBIN)/../api/"; \
+	go mod tidy; \
+	popd
```

Comment:
I don't think the pushd/popd are necessary. Make will execute the commands in its own shell, so you don't need to preserve the current directory. And since several other targets assume they're invoked from the top level directory, I think the following sequence should work:

.PHONY: tidy
tidy:
	go mod tidy
	cd ./api
	go mod tidy

Reply:
fyi you can use markdown formatting here so for code blocks you can use three of the  backticks (`) see https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-and-highlighting-code-blocks#fenced-code-blocks